README.md

```diff
@@ -108,130 +104,27 @@ automatically publish it.
 If you want to automatically release your packages for example when a new PR is
 merged you have a few different options:
 
-1. Create two actions, one which versions and dry-run publishes your packages
+1. **Version and release on workflow dispatch:**
+   Create two actions: one which versions and dry-run publishes your packages
+   and then creates a release preparation PR when trigger this workflow. 
+   And a second action that publishes your packages to pub.dev when the release
+   preparation PR is merged.
+2. **Version and release on each PR:**
+   Create two actions, one which versions and dry-run publishes your packages
    and then creates a release preparation PR every time a PR from a normal user
-   is merged and then a second action that publishes your packages to pub.dev 
+   is merged and then a second action that publishes your packages to pub.dev
    when the release preparation PR is merged.
-2. Directly publish your packages to pub.dev when a PR from a normal user is
+3. **Directly publish to pub.dev after user PR is merged:**
+   Directly publish your packages to pub.dev when a PR from a normal user is
    merged and create a PR with the versioning changes afterwards.
-3. Only use the action to ensure in your pipeline that your packages are
+4. **Only run dry-run versioning and publishing as a check:**
+   Only use the action to ensure in your pipeline that your packages are
    releasable, i.e doesn't fail any dry run.
 
 And this note is worth repeating - Remember to check the "Allow GitHub Actions
 to create and approve pull requests" checkbox in the bottom of the
 Actions > General section of your repository settings if you want to use
 `create-pr: true`.
 
-
-### 1. Version and release on each PR
-
-Create a file with the following content name `prepare-release.yaml` and place
-it in `.github/workflows/`:
-
-```yaml
-name: Prepare release
```

Comment:
It would be good to have some example files still in the readme, or references to them at least.

Reply:
Aah, just realized that this is a draft

Reply:
Ya still WIP 😅

Reply:
I added a link. For the default workflow, we would need 3 files. Idk if the README gets kind of heavy if copying them there.

Reply:
Yeah, like will do fine :)